src/api/index.js

```diff
@@ -227,3 +229,18 @@ export async function rejectApplication(mentor, reason) {
   );
   return res.success;
 }
+
+export async function sendMentorshipRequest(mentor, mentorshipRequest) {
+  const res = await makeApiCall(
+    `${paths.MENTORSHIPS}/${mentor._id}/apply`,
+    { message: mentorshipRequest.message },
```

Comment:
If we send only message, let's remove the other fields from the form. Or at least, comment out.